EIPS/eip-0000.md

```diff
@@ -0,0 +1,103 @@
+---
+eip: 0000
```

Comment:
```suggestion
eip: 7998
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.